src/views/catalog/wizard/tabs/overview/components/WizardOverviewDisksTable/WizardOverviewDisksTable.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+
+import useDisksTableData from '../../../../../../virtualmachines/details/components/disk/hooks/useDisksTableData';
```

Comment:
@yaacov as you can see I want to share logic with a hook that already does what I need because it tackled the problem before me, in this case getting a disk details.

This import is wrong, so we have two ways to tackle this:

1. Copy the code to my context, having to maintain duplicate code in multiple places (very bad IMO)
2. Create a shared utils/hooks folder for each resource (VM, VMI, Templates)